modules/sdk-core/src/bitgo/lightning/lightning.ts

```diff
@@ -59,8 +62,25 @@ export class Lightning implements ILightning {
     });
   }
 
-  public async withdraw(params?: LightningWithdrawalParams): Promise<WithdrawResponse> {
-    throw new Error('method not implemented');
+  public async withdraw(params: LightningWithdrawalParams): Promise<WithdrawResponse> {
+    const { value } = params;
+    let { destination, sequenceId } = params;
+
+    if (destination === undefined) {
+      destination = (await this.wallet.createAddress()).address;
+    }
+
+    if (sequenceId === undefined) {
+      sequenceId = `${Math.round(Math.random() * 1e10)}`;
```

Comment:
doesn't matter much in this case
```suggestion
      sequenceId = crypto.randomBytes(16).toString('hex');
```

Reply:
This seems a nicer way to generate `sequenceId`

Reply:
Thought about suggesting something similar. FWIW if we do go this route, 16 bytes is probably overkill since it only needs to be unique per wallet. 8 should be plenty. But that's a nit of a nit.